Detectors/MUON/MCH/Evaluation/CMakeLists.txt

```diff
@@ -33,3 +33,8 @@ o2_target_root_dictionary(MCHEvaluation
                             include/MCHEvaluation/Draw.h
                           LINKDEF src/MCHEvaluationLinkDef.h)
 
+o2_add_executable(map_mch
```

Comment:
`map_mch` is not a terribly good name :wink:
Especially considering the executable name will be `o2-mch-map_mch` which stutters on Mch and does not respect the executable naming convention.

What about something along the lines of `clustermap-display` or `clustermap-plotter` ? @mguilbau any opinion here ?

Reply:
Clustermap-display is my preferred